storage/innobase/include/fil0crypt.h

```diff
@@ -16,21 +16,21 @@ this program; if not, write to the Free Software Foundation, Inc.,
 
 *****************************************************************************/
 
-/**************************************************//**
-@file include/fil0crypt.h
-The low-level file system encryption support functions
+/**************************************************/ /**
```

Comment:
Reformatting diff looks interesting with so much commented out code. Can remove at least some of it?